FreeRTOS/FreeRTOS
# Build: Fix Demo/RX200_RX231-RSK_GCC_e2studio_IAR RTOS demo project IAR build error

Description
-----------
This PR fixes the following link error of Demo/RX200_RX231-RSK_GCC_e2studio_IAR RTOS demo IAR project by removing no longer necessary files at least using EWRX V4.14.1. Additionally, code for new evaluation board TB-RX231 is added to test this PR.

Link error messages:
```
Error[Ta001]: Mismatching linker configuration file version. Compare your file with the ready-made linker configuration files (.icf) in the rx\config directory! 
Error[Lp011]: section placement failed 
            unable to complete "place at" directives with a total estimated minimum size of 0xfc bytes in <[0xffff'ff80-0xffff'fffb], [0xffff'fffc-0xffff'ffff]> (total space 0x80). 
            Needed: 
            "A1" 
            [0xffff'ff80-0xffff'fffb]: 0xf8 min, align 0x1 (size: 0x7c) 
            "A2" 
            [0xffff'fffc-0xffff'ffff]:  0x4 min, align 0x1 (size:  0x4) 
Error[Lp015]: section placement failure: overcommitted content in [0xffff'ff80-0xffff'fffb] 
```
Cause of the error: Both compiler's default exceptvect.o and this project's exceptvect_rxv2.o are linked.

Fix of the error: The following files and folder are no longer necessary so that these files are removed.
```
src/IAR_Support/defaults.s
src/IAR_Support/exceptvect_rxv2.s
src/IAR_Support/option_rom_rxv2.s
src/IAR_Support/lnkr5f52318.icf
src/IAR_Support/
```
TB-RX231:

Target Board for RX family
[https://www.renesas.com/products/software-tools/boards-and-kits/eval-kits/rx-family-target-board.html](https://www.renesas.com/products/software-tools/boards-and-kits/eval-kits/rx-family-target-board.html)

Test Steps
-----------
Both full_demo and blinky_demo are tested to be run expectedly. TB-RX231 is used by defining 'TB_RX231' on EWRX as following screen copy. SREC file is burn into MCU on TB-RX231 using RFP(Renesas Flash Programmer) software via on-board E2 Lite emulator hardware. But I don't have any RSK-RX231 so that programs aren't tested using any RSK-RX231.

![e2studio_freertos_build_issues_91](https://user-images.githubusercontent.com/22914520/92046919-9fdf4480-edbe-11ea-823b-e2b192020b17.png)

Related Issue
-----------
None.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
